### DISCUSSION TITLE
Is it possible to set tokenizer path and hf checkpoint path separately when converting to gguf?

### DISCUSSION THREAD

**Discussion by lidh15** [ggml-org/llama.cpp#8312] [Q&A]
In my experiments, I have lots of different checkpoints that I wanna try, and of course they are using the same tokenizer. In pytorch, it is not a problem to set a static tokenizer path and investigate different checkpoints, but when trying to convert to gguf with llama.cpp's script, it always wanna see the tokenizer in the same directory as the checkpoint and I have to copy the tokenizer once and once again. why not provide the converter an optional arg and let the same dir as a default option?

**Comment by ggerganov** [comment]
It hasn't been implemented yet - PRs welcome
